### DISCUSSION TITLE
How traditional Java app use apollo? 传统java项目怎么集成apollo

### DISCUSSION THREAD

**Discussion by MMCISAGOODMAN** [apolloconfig/apollo#3440] [Q&A]
传统java项目怎么集成apollo,非spring项目，可以给一个demo吗

**Comment by klboke** [comment]
可以参考这个项目，非spring环境的，纯apollo client集成quarkus配置接口实现：https://github.com/kekingcn/quarkus-apollo-config

**Comment by MMCISAGOODMAN** [comment]
thanks


在2020年12月19日 21:35，kl<notifications@github.com> 写道：

可以参考这个项目，非spring环境的，纯apollo client集成quarkus配置接口实现：https://github.com/kekingcn/quarkus-apollo-config
